html/demos/streaming.js

```diff
@@ -346,13 +355,9 @@ function updateStreamsList() {
 				// Keep track of all the available streams
 				streamsList[list[mp]["id"]] = list[mp];
 			}
-			$('#streamslist a').unbind('click').click(function() {
-				$('.dropdown-toggle').dropdown('hide');
+			$('#streamslist a').on("click", function() {
```

Comment:
We had an explicit unbind because this code is called any time the list of streams is refreshed. I'm not sure how `on` works, but will this replace the previous handler, or cause multiple callbacks to be registered, and the same function being called more than once?

Reply:
Thank you for checking.
I add .off('click').